### PR TITLE
Pipelines in corefx

### PR DESCRIPTION
**Pull request by pakrym**
Disregard source.props

### CODE REVIEW

File: samples/SocketsSample/EndPoints/MessagesEndPoint.cs [hunk 1]
```diff
@@ -64,10 +65,11 @@ private Task Broadcast(string text)
         private Task Broadcast(byte[] payload)
         {
             var tasks = new List<Task>(Connections.Count);
+            async Task<FlushResult> ToTask(PipeAwaiter<FlushResult> awaiter) => await awaiter;
 
             foreach (var c in Connections)
             {
-                tasks.Add(c.Transport.Output.WriteAsync(payload));
+                tasks.Add(ToTask(c.Transport.Output.WriteAsync(payload)));
```
Comment (mikaelm12) [inline samples/SocketsSample/EndPoints/MessagesEndPoint.cs]: This is a little weird. Why exactly is this happening

File: src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsTransport.cs [hunk 2]
```diff
@@ -93,7 +93,7 @@ public Task StartAsync(Uri url, IDuplexPipe application, TransferMode requestedT
                     {
                         var result = await pipelineReader.ReadAsync();
                         var input = result.Buffer;
-                        if (result.IsCancelled || (input.IsEmpty && result.IsCompleted))
+                        if (result.IsCanceled || (input.IsEmpty && result.IsCompleted))
```
Comment (mikaelm12) [inline src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsTransport.cs]: There was talk in the demo meeting about a new API for something like HasData (I don't remember what was actually said). Coming later?

Comment (pakrym) [reply src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsTransport.cs]: Definitely not in this iteration.

File: samples/SocketsSample/EndPoints/MessagesEndPoint.cs (hunk 1 above)
Comment (pakrym) [reply samples/SocketsSample/EndPoints/MessagesEndPoint.cs]: WriteAsync returns custom Awaiter object, not Task that the list requires.

**Pull request merged**